doc/level1.rst

```diff
@@ -67,24 +67,21 @@ encryption is possible, and:
 Requirements on MUA/E-mail Provider interactions
 ++++++++++++++++++++++++++++++++++++++++++++++++
 
-Autocrypt tries to impose minimal requirements on how :term:`MUA` s and
-e-mail services interact.  We assume that an Autocrypt-capable MUA
-has credentials and capabilities to perform these network services:
+Autocrypt tries to impose minimal requirements on :term:`MUA` and
+e-mail service interactions.  Specifically, an Autocrypt-capable MUA
+needs to be able to:
 
-- The ability to send e-mail (e.g. via SMTP or Submission) where the
-  MUA can control the entire message being sent, including both
-  message headers and message body.
+- Control the contents of outgoing e-mail including the ability to set
+  custom e-mail headers;
```

Comment:
this drops the requirement to send messages on its own, which is required for the setup message. I still like the change, maybe we can just move sending messages to a different point

Reply:
ack.